packages/contracts-bedrock/contracts/universal/Proxy.sol

```diff
@@ -121,7 +121,7 @@ contract Proxy {
      *
      * @return Owner address.
      */
-    function admin() external proxyCallIfNotAdmin returns (address) {
```

Comment:
This is currently designed in a way where we do not need to worry about function selector clashing. This change will result in the potential for function selector clashing